Update Pricing.md (#4870)
SkylineCommunications/dataminer-docs: dataminer-overview/Pricing/Professional_service_credits.md

## Engineering

| Item | Details |
|------|---------|
| Integration | 4 credits<br>Per hour, per person |
| Consultancy | 5 credits<br>Per hour, per person |

## Certifications

| Item | Details |
|------|---------|
| Fundamentals Certificate | 16 credits<br>Per certificate |
| Configurator Certificate | 36 credits<br>Per certificate |
| Visual Overview | 6 credits<br>Per certificate |
| Automation Certificate | 24 credits<br>Per certificate |
| Connector Integration: SNMP Basics | 40 credits<br>Per certificate |
| Connector Integration: HTTP Basics | 40 credits<br>Per certificate |

## Dedicated training

| Item | Details |
|------|---------|
| DevOps Enablement Service | 5 credits<br>Per hour, per person |
| [Open Day Training](https://community.dataminer.services/open-training/) | 14 credits<br>Per day, per person |
| Fundamentals Training (2 days) | 160 credits<br>Per session |
| Configurator Training (2 days) | 160 credits<br>Per session |
| Sys Admin Training (1 day) | 90 credits<br>Per session |
| Agile Fundamentals Training | 40 credits<br>Per session |
| Kanban Workshop | 40 credits<br>Per session |

> [!NOTE]
>

> > - Except for Agile Awareness and DevOps Enablement, all training is available in both in-person and online format. For in-person training, additional credits are required to cover the transportation and on-site expenses of having the instructor present. These expenses are not included in the course fees.

Reviewer: Agile Awareness is not mentioned in the table above. Should I change this to something else, or is something missing?

> > - The number of hours required by a program depends on factors like the number of people, their skills, their profile, and the desired program outcome.

Reviewer: This note was linked to a footnote marker next to the DevOps Enablement Service item; however, the way it's phrased suggests that it might not be limited to that item. Could you double-check what this refers to?

Author: This note should be linked to the header '_dedicated training_'. This means that all training below the header '_dedicated training'_ are available both in-person and online format except for the DevOps Enablement Service. Agile Fundamentals is also available now in both formats.

Reviewer: I've updated line 46 now. Just to be sure: line 47 is OK as it is?

Author: Line 47 is related to the DevOps Enablement Service and is ok

Author: https://community.dataminer.services/professional-service-credits/

Reviewer: If line 47 is related to the DevOps Enablement Service, I'd change this as follows:
```suggestion
> - The number of hours required for the DevOps Enablement Service depends on factors like the number of people, their skills, their profile, and the desired program outcome.
```

Author: Looks good. Thanks
